### ISSUE TITLE
Installation fails on Ubuntu 16.04

### ISSUE THREAD

**Issue by c-geek** [duniter/sakia#453]
Using Gnome UI, I click on the "sakia.deb" file downloaded from GitHub, but the installation seems to be waiting forever.

If I install it with `sudo dpkg -i sakia.deb` then I can run Sakia, but then the Sakia UI never shows up. I get stuck on `LOADER: Running main.py`.

**Comment by c-geek** [comment]
OK Sakia does start, but the *very first start* is very long and I can see the following messages:

```text
LOADER: Running main.py
/home/cgeek/Téléchargements/dist/sakia/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /usr/lib/x86_64-linux-gnu/libproxy.so.1)
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiolibproxy.so

(sakia:2047): Gtk-WARNING **: GModule (/usr/lib/x86_64-linux-gnu/gtk-2.0/2.10.0/immodules/im-ibus.so) initialization check failed: GLib version too old (micro mismatch)

(sakia:2047): Gtk-WARNING **: Loading IM context type 'ibus' failed

(sakia:2047): Gtk-WARNING **: GModule (/usr/lib/x86_64-linux-gnu/gtk-2.0/2.10.0/immodules/im-ibus.so) initialization check failed: GLib version too old (micro mismatch)

(sakia:2047): Gtk-WARNING **: Loading IM context type 'ibus' failed

(sakia:2047): Gtk-WARNING **: GModule (/usr/lib/x86_64-linux-gnu/gtk-2.0/2.10.0/immodules/im-ibus.so) initialization check failed: GLib version too old (micro mismatch)

(sakia:2047): Gtk-WARNING **: Loading IM context type 'ibus' failed
```

Next starts behave normally, Sakia is launched in under few seconds.

**Comment by Insoleet** [comment]
Well the deb files are built on Ubuntu 14.04 so there are probably some incompatibilities here...

**Comment by Insoleet** [comment]
On the chat you said it was fixed with ubuntu upgrade ? Can we close this ?

**Comment by c-geek** [comment]
Yes we can close because:

* the installation fails due to Ubuntu: updating the OS with the latest updates fix the issue
* the launching delay can be fixed by #454
